tsconfig.json

```diff
@@ -11,46 +11,31 @@
     },
     "types": ["node"],
     "exclude": [
-        "./data/mods/gen1/pokedex.js"
+        "./data/mods/gen1/pokedex.js",
+        "./server/index.js",
```

Comment:
Why does `index` need to be excluded?

Reply:
Because if I don't exclude it `tsc` will report 451 errors... Try for yourself :)